### PR TITLE
Add title attribute to media player

### LINKED ISSUE #2403: Add title attribute to the Media Player component
**Is your feature request related to a problem? Please describe.**
To enable us to do the work in https://github.com/bbc/simorgh/issues/3961 we need to be able to pass in a `title` prop to the `psammead-media-player` which is then used as an attribute on the `<iframe>`.

**Describe the solution you'd like**
Accept a `title` prop in the Media Player and use it's value to populate the `<iframe>`'s `title` attribute. 

**Describe alternatives you've considered**
N/a

**Testing notes**
[Tester to complete]

Dev insight: Snapshots and storybook stories are required. 

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #2403

**Overall change:** 
Add a title prop which is passed down to both amp and canonical iframes

**Code changes:**
- Add title attribute to players
- Update tests
- Update stories

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-media-player/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version       | Description                                                                                                                  |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| 1.1.1-alpha.8 | [PR#2434](https://github.com/bbc/psammead/pull/2434) Add `title` attribute to AV media player |
```
Comment (pjlee11) [inline packages/components/psammead-media-player/CHANGELOG.md]: We can remove alpha from this component now that title is supplied

File: packages/components/psammead-media-player/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-media-player",
-  "version": "1.1.1-alpha.7",
+  "version": "1.1.1-alpha.8",
   "publishConfig": {
```
Comment (pjlee11) [inline packages/components/psammead-media-player/package.json]: Remove `publishConfig` to remove alpha tag.

**Review by ryanmccombe — APPROVED** [review]

File: packages/components/psammead-media-player/CHANGELOG.md (hunk 1 above)
Comment (pjlee11) [reply packages/components/psammead-media-player/CHANGELOG.md]: Also as we are adding the new required prop of `title` this should be a major change IMO to `2.0.0`

**Review by pjlee11 — CHANGES_REQUESTED** [review]
Couple of small tweaks needed

**Comment by pjlee11** [comment]
Just removing `-alphaX` from the version number doesn't stop npm publishing it as a alpha, you need to remove the following from the package.json and then install to update the lock
```
  "publishConfig": {
    "tag": "alpha"
  },
```

**Comment by pjlee11** [comment]
Could you update the READMEto include the `title` prop for both AMP and canonical. Also mention in `Accessibility notes` that the title is used for a11y best practice on the iframe.

**Review by pjlee11 — APPROVED** [review]
Nice work 😎

**Review by ibMadbouly — APPROVED** [review]
LGTM 👍

**Review by chris-hinds — APPROVED** [review]

**Comment by EinsteinNjoroge** [comment]
You might wanna check some of these boxes in the description. 
Screen Shot 2019-10-18 at 4 49 40 PM

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**